zio-schema/shared/src/main/scala/zio/schema/Schema.scala

```diff
@@ -393,11 +403,16 @@ object Schema extends SchemaEquality {
 
   final case class Tuple[A, B](left: Schema[A], right: Schema[B], annotations: Chunk[Any] = Chunk.empty)
       extends Schema[(A, B)] { self =>
-    override type Accessors[Lens[_, _], Prism[_, _], Traversal[_, _]] = (Lens[(A, B), A], Lens[(A, B), B])
+
+    val first = "_1"
+    val second = "_2"
```

Comment:
I am using these strings as singleton type parameters. If I inline them it fails to compile on 2.12